### PR TITLE
add tensor dilation parameter configuration for acl depthwise conv

### PR DESCRIPTION
**Pull request by snadampal**
updated the existing onednn acl depthwise conv patch in pytorch docker to add tensor dilation parameter configuration.

### CODE REVIEW

**Review by nSircombe — COMMENTED** [review]
Thanks for the PR.
Couple of questions...
- Dilation support would also require an increment to the ACL version used in the build (https://github.com/ARM-software/ComputeLibrary/issues/1051). Is that right?
- The additional capability should be noted in the CHANGELOG.md under "Unreleased", "Added".

**Comment by snadampal** [comment]
yes, for depthwise convolution dilation feature to work, it needs the ACL change, but having this change without the ACL doesn't break the build.  I think once it is merged, a single PR can be raised against oneDNN for depthwise conv support.

**Comment by nSircombe** [comment]
> for depthwise convolution dilation feature to work, it needs the ACL change, but having this change without the ACL doesn't break the build.

We'll have to look at the practicality of back-porting ACL dilation support into this build ahead of merging this PR.

> a single PR can be raised against oneDNN for depthwise conv support.

This would have to wait until after the next Compute Library release though. It will introduce a dependency on ACL 23.05, and will need the min. ACL version that oneDNN checks for to be incremented accordingly.

Even if the oneDNN _build_ is not impacted, I don't believe it would be able to pass the `benchdnn` suite of tests, as these have a number of batches for `conv` with dilation, which I would expect

**Comment by nSircombe** [comment]
Hi @snadampal,
I'm testing this with our existing test cases now to check nothing's broken by the change, but there are a few conflicts now that need to be resolved before this can be merged.

**Comment by nSircombe** [comment]
@snadampal  -  there were not issues with any of the testing I've done. If you can resolve the conflicts (it's all minor stuff) and confirm that the build does what you're expecting with your test case then I will merge.

File: docker/tensorflow-aarch64/CHANGELOG.md
Comment (nSircombe) [inline docker/tensorflow-aarch64/CHANGELOG.md:23]: This CHANGELOG change is in the wrong place - these changes aren't part of the 23.05 increment. They should go in the [Unreleased] section initially, and will then be included in the changes for the next release.

File: docker/pytorch-aarch64/CHANGELOG.md
Comment (nSircombe) [inline docker/pytorch-aarch64/CHANGELOG.md:23]: This CHANGELOG change is in the wrong place - these changes aren't part of the 23.05 increment. They should go in the [Unreleased] section initially, and will then be included in the changes for the next release.

**Review by nSircombe — APPROVED** [review]
@snadampal - I made some trivial changes to the CHANGELOG.md's.

**Comment by snadampal** [comment]
Thanks @nSircombe !

**Pull request merged**
